### DISCUSSION TITLE
C'est quoi @Input ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#7] [General]
# C'est quoi @Input ? | Bonjour Angular

C'est quoi @Input dans Angular ?

https://apprendre-dev.vercel.app/cest-quoi/input/

**Comment by ben-barbier** [comment]
Merci pour ces exemples !

Pourrais-tu expliquer dans les `Route Input` d'où vient le `foo` et l'afficher dans le template ?

Merci 🚀 !

**Reply by KevTale** [reply]
C'est fait ! `foo` vient du queryParam :)

**Comment by ph1823** [comment]
Le liens vers les signaux avec input pointe vers un 404 :( il arrive bientôt ? 😁

**Comment by vnivuahc** [comment]
Je crois que le bon lien c’est : https://apprendre.bonjour-angular.com/videos/signal-inputs/
